modules/renter/hostdb/scan.go

```diff
@@ -107,10 +107,10 @@ func (hdb *HostDB) decrementReliability(addr modules.NetAddress, penalty types.C
 
 	// If the entry is in the active database, remove it from the active
 	// database.
-	node, exists := hdb.activeHosts[addr]
+	existingEntry, exists := hdb.activeHosts[addr]
```

Comment:
so the entries still have the reliability cruft, instead of using the scan data?